stylecop.json

```diff
@@ -36,14 +36,14 @@
       "companyName": ".NET Foundation",
       "copyrightText": "Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.\nSee the LICENSE file in the project root for more information",
       "xmlHeader": false,
-      "documentInterfaces": true,
-      "documentExposedElements": true,
-      "documentInternalElements": true,
+      "documentInterfaces": false,
+      "documentExposedElements": false,
+      "documentInternalElements": false,

```

Comment:
I took the liberty of changing this as the code would never be part of our public API. In the interest of time I wanted to advance rather than spend my time documenting code.